Makefile

```diff
@@ -1,4 +1,16 @@
 SHELL = /bin/bash
+TAG ?= $(shell git describe --exact-match 2>/dev/null)
+COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
+# There is currently no ACR for ARO HCP components. Variable will be defined later
+ARO_HCP_BASE_IMAGE = ${ARO_HCP_IMAGE_ACR}.azurecr.io/aro
```

Comment:
i think we want to drop the `/aro` on this.  ACRs aren't set up like organizations are typically in quay. 

We don't want to accidentally write over the `aro` image

Reply:
```suggestion
ARO_HCP_BASE_IMAGE = ${ARO_HCP_IMAGE_ACR}.azurecr.io